src/components/bikeTable/BikeTable.jsx

```diff
@@ -3,13 +3,17 @@ import { getIcon } from '../../utils'
 import './bikeTable.css'
 
 const BikeTable = ({ stationData, visible }) => {
+    const { hiddenModes, hiddenStations } = visible
     const stations = stationData
-        .filter(({ id }) => !visible.includes(id))
+        .filter(({ id }) => !hiddenStations.includes(id))
         .sort((a, b) => a.name.localeCompare(b.name))
+    if (hiddenModes.includes('bike')) {
```

Comment:
Denne kan flyttast opp til før `const stations = ...` for å optimalisere bittelittegrann.